pom.xml

```diff
@@ -42,6 +42,13 @@
             <artifactId>appengine-api-1.0-sdk</artifactId>
             <version>1.9.59</version>
         </dependency>
+
+        <dependency>
```

Comment:
Sydney put the soy dependency as the first one in the dependencies tag and that already merged to master, so you might want to move this to the top just in case you get a merge conflict